drivers/display/Kconfig.mcux_dcnano_lcdif

```diff
@@ -0,0 +1,56 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+
+menuconfig DISPLAY_MCUX_DCNANO_LCDIF
+	bool "MCUX DCNano LCDIF driver"
+	default y
+	depends on DT_HAS_NXP_DCNANO_LCDIF_ENABLED
+	help
+	  Enable support for mcux DCNano LCDIF driver.
+
+
+if DISPLAY_MCUX_DCNANO_LCDIF
+
+config MCUX_DCNANO_LCDIF_DOUBLE_FRAMEBUFFER
+	bool "Double framebuffer"
+	help
+	  Enable dual framebuffer for LCDIF peripheral. Two framebuffers
+	  will be allocated, and switched between for each frame.
+	  Note that for partial display updates, the prior framebuffer must
+	  be copied into the next one. This can have significant performance
+	  impact.
+
+config MCUX_DCNANO_LCDIF_MAINTAIN_CACHE
```

Comment:
I'm adding my comment as another potential PR suggestion.  This MAINTAIN_CACHE concept is not specific to the LCDIF.  In this RT500 family, the cache is on the AHB slave port of the FlexSPI memory interface.  This means all masters accessing the FlexSPI share this common cache, and therefore coherency is not an issue.  Other drivers could use this feature as well.  Perhaps a better place for this feature is a DT property for the MEMC device.  And if the LCDIF or other drivers are accessing a MEMC device with MAINTAIN_CACHE property, then cache management is not required.  But we do not need to delay this PR to implement that.  Let's consider for a future improvement.